src/Microsoft.DotNet.Interactive/Commands/RequestInput.cs

```diff
@@ -5,14 +5,17 @@ namespace Microsoft.DotNet.Interactive.Commands;
 
 public class RequestInput : KernelCommand
 {
-    public RequestInput(string prompt, bool isPassword = false, string targetKernelName = null)
+    public RequestInput(string prompt, bool isPassword = false, bool isConnectionString = false, string targetKernelName = null)
```

Comment:
Do you think you'll have other types of input? (bool, int, enum, etc.) If so may want to look into having this all consolidated into an `inputType` param instead